components/bl-list-component/src/list.js

```diff
@@ -0,0 +1,25 @@
+export function List({ options, items, type }) {
+  const ListElement = type || options.type;
+
+  return (
+    <ListElement className="list">
+      { items.map((item, i) => {
+        const { content, children, type } = item;
+
+        return <Item key={ i } content={ content } children={ children } type={ type } options={ options } />;
+      }) }
+    </ListElement>
+  );
+}
+
+function Item(props) {
+  const { options, key, content, children, type } = props;
+  const { style, onClickHandler } = options;
```

Comment:
can we avoid using the `options`?

Reply:
I think we can't, this parameter is needed both for the list and for the item inside.